index.html

```diff
@@ -0,0 +1,131 @@
+            <h2 class="main-text">Main Program</h2>
+            <div class="seperate-line"></div>
+          </div>
+          <div class="program-container">
+            <div class="program-1">
+              <img src="images/program_icon_01.svg" alt="program-icon" class="program-img">
+              <h3 class="program-text">Lecture</h3>
+              <p class="program-overview-text">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="program-2">
+              <img src="images/program_icon_02.svg" alt="program-icon" class="program-img">
+              <h3 class="program-text">Ehibition</h3>
+              <p class="program-overview-text">Appreciate various creations applying Microsoft license of developers, organized Microsoft</p>
+            </div>
+            <div class="program-3">
+              <img src="images/program_icon_03.svg" alt="program-icon" class="program-img">
+              <h3 class="program-text">Forum</h3>
+              <p class="program-overview-text">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            </div>
+            <div class="program-4">
+              <img src="images/program_icon_04.svg" alt="program-icon" class="program-img">
+              <h3 class="program-text">Workshop</h3>
+              <p class="program-overview-text">Try creating your own work using open source license rather than just watching at it.</p>
+            </div>
+            <div class="program-5">
+              <img src="images/program_icon_05.svg" alt="program-icon" class="program-img">
+              <h3 class="program-text">Ignite</h3>
+              <p class="program-overview-text">get opportunities to network with Microsoft affiliates around the world, also after the summit.</p>
+            </div>
+          </div>
+          <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <!--section 3 built dynamically-->
+      <section class="section-3">
+        <div class="sec-3-head">
+          <h2 class="sec-3-header">Featured speakers</h2>
+          <div class="sec-3-line"></div>
+        </div>
+        <div class="sec-3-container">
+
+        </div>
+      </section>
+      <!--section 3 built dynamically-->
+      <section class="section-4">
+        <div class="patner-header">
+          <h2 class="patner-title">Patner</h2>
+          <div class="seperate-line-2"></div>
+        </div>
+        <div class="sponsors">
+          <img src="images/dam.svg" alt="company-logos" class="company-logos">
+          <img src="images/mozilla 1.svg" alt="company-logos" class="company-logos">
+          <img src="images/air.svg" alt="company-logos" class="company-logos">
+          <img src="images/naver 1.svg" alt="company-logos" class="company-logos">
+          <img src="images/gog.svg" alt="company-logos" class="company-logos">
+        </div>
```

Comment:
- [ ] As pointed out by the previous reviewer, you can see that not all the speakers are shown on mobile. It shows only two with a `more button`, I'll suggest you implement this on your project. :clap: 

- - Your design:
![Screenshot from 2023-06-30 09-21-24](https://github.com/2004-okari/CAPSTONE-01/assets/107632780/944f4bf6-854b-4eab-9fd1-668bd20cee60)

- - Given design:
![Screenshot from 2023-06-30 09-21-55](https://github.com/2004-okari/CAPSTONE-01/assets/107632780/699e606d-2e80-45b9-9163-c9391a714968)

- [ ] Also as the previous reviewer pointed out, the `Partner and the footer` section are not visible on mobile screens according to the design. I'll suggest you update your CSS to hide these sections on mobile and display them when on desktop screens. :smile: :+1: